src/raise.cr

```diff
@@ -175,16 +175,21 @@ fun __crystal_get_exception(unwind_ex : LibUnwind::Exception*) : UInt64
   unwind_ex.value.exception_object
 end
 
-def raise(ex : Exception) : NoReturn
-  ex.callstack = CallStack.new
+# Raises the *exception*.
+#
+# Note that exception callstack will only be set if it's not already set.
+# It's useful for reraising of exceptions in `rescue` blocks.
```

Comment:
I suppose that **the** article just fit here.

And thanks for rewording, will apply.